### DISCUSSION TITLE
Rename an Item

### DISCUSSION THREAD

**Discussion by tobru** [TomBursch/kitchenowl#131] [Q&A]
How can I rename / edit an item which is already available in the database?

**Comment by TomBursch** [comment]
Edit: you can now rename an item in the app.

---

There is currently no way to do that. In #121 I said that unless many people want this feature, I don't think I'll add it. The easiest way is to go through all recipes that include the item you want to rename, replace it with a new one (you can see all recipes with a specific item by long clicking that item), and then delete the old item (also by long clicking it and then delete in the top right).

Advanced method: you could edit the item name in the database directly (at your own risk) with a tool like https://sqlitebrowser.org/.

**Reply by tobru** [reply]
Thanks, I might go the database edit way then.
